### PR TITLE
Updated swipl stable to 9.2.5

### PR DESCRIPTION
**Pull request by JanWielemaker**

### CODE REVIEW

File: library/swipl [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Maintainers: Jan Wielemaker <jan@swi-prolog.org> (@JanWielemaker),
              Dave Curylo <dave@curylo.org> (@ninjarobot)
 GitRepo: https://github.com/SWI-Prolog/docker-swipl.git
-GitCommit: 3ef22f678af652268d24285defc0ae60e62e6d26
+GitCommit: e5e2a61337a7f21d8b6c53fb6521d507b9d48771
```
Comment (whalelines) [inline library/swipl:4]: It seems this commit does not exist in the SWI-Prolog/docker-swipl repository. This gives a 404: https://github.com/SWI-Prolog/docker-swipl/commit/e5e2a61337a7f21d8b6c53fb6521d507b9d48771

**Review by whalelines — COMMENTED** [review]
The checks are failing, seemingly because the referenced commit does not exist.

File: library/swipl (hunk 1 above)
Comment (JanWielemaker) [reply library/swipl:4]: > It seems this commit does not exist in the SWI-Prolog/docker-swipl repository

Thanks.   Yes, forgot to push :cry:    Can you restart the checks or should I delete this PR and recreate exactly the same?

Comment (whalelines) [reply library/swipl:4]: The jobs have been restarted.

**Review by yosifkit — APPROVED** [review]

**Pull request merged**